### PR TITLE
added Depth and Perspective Visualizer

### PR DESCRIPTION
**Pull request by abckhush**
Fixes: #155

### CODE REVIEW

**Comment by abckhush** [comment]
@dinxsh Kindly review this PR. It's been a week

**Review by dinxsh — CHANGES_REQUESTED** [review]
lgtm. could you add a screenshot of the output. sorry for the delayed review!

**Comment by abckhush** [comment]
image
image

**Review by dinxsh — APPROVED** [review]
nice, good work

**Comment by abckhush** [comment]
@Soumya-Kushwaha Kindly review this.

**Review by Soumya-Kushwaha — APPROVED** [review]

**Comment by abckhush** [comment]
Kindly merge this.

**Review by dinxsh — CHANGES_REQUESTED** [review]
there's a conflict now, could you check @abckhush

**Comment by abckhush** [comment]
@dinx check

File: mainLanding.py [hunk 1]
```diff
@@ -141,8 +142,9 @@ def launch_visualizer(self, instance):
         visualizer_scripts = {
             "Amplitude-Frequency Visualizer": "Amplitude-Frequency-Visualizer.py",
             "Waveform": "Waveform.py",
-            "Spectrogram": "Spectrogram.py",
-            "Intensity vs Frequency and Time": "Intensity-vs-Frequency-and-Time.py"
+            "Spectrogram": "Spectogram.py",
```
Comment (dinxsh) [inline mainLanding.py:145]: Supposed to be Spectrogram.py

**Review by dinxsh — CHANGES_REQUESTED** [review]

**Comment by abckhush** [comment]
@dinxsh done.

**Review by dinxsh — APPROVED** [review]
good

**Pull request merged**